### DISCUSSION TITLE
[ FF7 ] Crash on 1 battle

### DISCUSSION THREAD

**Discussion by seito10** [julianxhokaxhiu/FFNx#290] [Q&A]
Using 7th Heaven 2.2 Crash everytime on 1st battle with or without mods enabled

"Oops! Something very bad happened.

Wrote emergency save to save/crash.ff7 dir

Please provide a copy of those files along with FFNx.LOG when reporting this error at 
https://github.com/julianxhokaxhie/FFNx/issues

**Expected behavior**
Able to play

**Screenshots**
![image](https://user-images.githubusercontent.com/77245016/138088842-ab6a6890-777f-40c8-b4ae-632f9aca4207.png)

**GPU (please complete the following information):**

- Brand NVidia RTX 3080
- Driver Version 3.23.0.74
[FFNx.log](https://github.com/julianxhokaxhiu/FFNx/files/7380996/FFNx.log)

**Comment by eve-atum** [comment]
Both your nVidia and FFNx drivers are way out of date.  Update and provide new logs (assuming it still crashes)

**Comment by seito10** [comment]
my bad it's the geforce experience i was looking at. my driver version is 471.96

**Comment by eve-atum** [comment]
That's two months back on nVidia, but at least it's much less likely to be the problem.  Still need to update your FFNx though; 1.7.2.124 is over a year old, and there's a huge amount of stuff that's dependent on the current canary builds.

**Comment by seito10** [comment]
how can I update my FFNx?

**Comment by eve-atum** [comment]
https://github.com/julianxhokaxhiu/FFNx/wiki/How-To_Install

**Comment by seito10** [comment]
thanks
